### PR TITLE
Add support or internal deposit address

### PR DESCRIPTION
**Pull request by ashish-nikalje**
## What's done
   - added api support for get platform deposit address for user
   - add api support for get platform user token balance
## note
this should be get merged after  https://github.com/TuringAdvisoryGroup/roll-go/pull/3700

## Tasks

- [ ] Have you written tests for new code (if applicable)?
- [ ] Have you tested the changes on all the platforms?
  - [ ] iOS
  - [ ] Android
  - [ ] Web
- [ ] Have you added stories to demonstrate the new functionality (if there is any)?

### CODE REVIEW

File: packages/api/src/user/user.ts [hunk 1]
```diff
@@ -77,6 +80,31 @@ export const createPlatformUser = async (
...
+  return response.data
+}
+
+export const getPlatformUserDepositAddress = async (
+  client: Client,
+  { userType, externalUserId }: ExternalUserArgs,
+) => {
+  const response = await client.call<Response<PlatformUserDepositAddressResponseData>>({
+    url: `/v1/platforms/${userType}/users/${externalUserId}/address`,
+    method: 'get',
```
Comment (diegonzs-roll) [inline packages/api/src/user/user.ts:92]:
```suggestion
    method: 'GET',
```

Comment (diegonzs-roll) [inline packages/api/src/user/user.ts:92]:
```suggestion
    method: 'GET',
```

File: packages/api/src/user/user.ts [hunk 2]
```diff
@@ -77,6 +80,31 @@ export const createPlatformUser = async (
...
+  return response.data
+}
+
+export const getPlatformUserBalance = async (
+  client: Client,
+  { userType, tokenId, externalUserId }: PlatformUserArgs,
+) => {
+  const response = await client.call<Response<PlatformUserBalanceTokenResponseData[]>>({
+    url: `/v1/platforms/${userType}/users/${externalUserId}/balances/${tokenId}`,
+    method: 'get',
```
Comment (diegonzs-roll) [inline packages/api/src/user/user.ts:105]:
```suggestion
    method: 'GET',
```

File: packages/sdk/src/auth/sdk.ts [hunk 3]
```diff
@@ -54,6 +54,7 @@ class SDK {
         refreshToken: this.token!.refresh_token!,
         code,
       })
+      console.log('response', response);
```
Comment (diegonzs-roll) [inline packages/sdk/src/auth/sdk.ts:57]: You can remove this console.log :)

**Review by diegonzs-roll — CHANGES_REQUESTED** [review]
Great work here @ashish-nikalje

**Review by diegonzs-roll — APPROVED** [review]

**Comment by marlon-wiprud** [comment]
closing in favor of the updates made here which avoided some of the conflicts in this PR: https://github.com/TuringAdvisoryGroup/tryrolljs/pull/311

**Pull request closed without merging**